### PR TITLE
[release-19.0] Release of `v19.0.0`

### PR DESCRIPTION
**Pull request by systay**
Includes the release notes and release commit for the `v19.0.0` release. Once this PR is merged, we will be able to tag `v19.0.0` on the merge commit.

### CODE REVIEW

**Review by frouioui — APPROVED** [review]

**Comment by frouioui** [comment]
Rebased this PR to include the PRs that have been merged during code freeze:
- https://github.com/vitessio/vitess/pull/15362
- https://github.com/vitessio/vitess/pull/15377
- https://github.com/vitessio/vitess/pull/15390
- https://github.com/vitessio/vitess/pull/15364

File: changelog/19.0/19.0.0/release_notes.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Vitess will however, continue to support importing from MySQL 5.7 into Vitess ev
 #### <a id="vttablet-flags"/>VTTablet Flags
 
 - The following flags — which were deprecated in Vitess 7.0 — have been removed:
-  `--vreplication_healthcheck_topology_refresh`, `--vreplication_healthcheck_retry_delay`, and `--vreplication_healthcheck_timeout`.
+`--vreplication_healthcheck_topology_refresh`, `--vreplication_healthcheck_retry_delay`, and `--vreplication_healthcheck_timeout`.
```
Comment (GrahamCampbell) [inline changelog/19.0/19.0.0/release_notes.md:51]: Indentation level changed in error?

File: changelog/19.0/19.0.0/release_notes.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ mysql> show vschema keyspaces;
 
 #### <a id="fk-checks-vitess-aware"/>`FOREIGN_KEY_CHECKS` is now a Vitess Aware Variable
 
-When VTGate receives a query to change the `FOREIGN_KEY_CHECKS` value for a session, instead of sending the value down to MySQL, VTGate now keeps track of the value and changes the queries by adding `SET_VAR(FOREIGN_KEY_CHECKS=On/Off)` style query optimizer hints wherever required.
+When VTGate receives a query to change the `FOREIGN_KEY_CHECKS` value for a session, instead of sending the value down to MySQL, VTGate now keeps track of the value and changes the queries by adding `SET_VAR(FOREIGN_KEY_CHECKS=On/Off)` style query optimizer hints wherever required.
```
Comment (GrahamCampbell) [inline changelog/19.0/19.0.0/release_notes.md:161]: Trailing space added in error?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by frouioui** [comment]
Rebased https://github.com/vitessio/vitess/pull/15401.

File: changelog/19.0/19.0.0/release_notes.md (hunk 1 above)
Comment (frouioui) [reply changelog/19.0/19.0.0/release_notes.md:51]: From what I am seeing it doesn't affect the format on GitHub.

**Review by shlomi-noach — APPROVED** [review]

**Pull request merged**